src/controllers/moviesController.js

```diff
@@ -0,0 +1,40 @@
+const { statusCode } = require("../helpers/constants.js");
+const {
+  getMovies,
+  getMovieById,
+} = require("../services/moviesService.js");
+
+const getAllMovies = async (req, res, next) => {

```

Comment:
You can omit using next fucntion, but wou have to use asyncWrapper in the appropriate router